docs/src/app/components/pages/components/popover.jsx

```diff
@@ -87,6 +87,16 @@ let PopoverPage = React.createClass({
       <ComponentDoc
         name="Popover"
         componentInfo={componentInfo}>
+        
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nimport Popover from \'material-ui/lib/popover\';\n\n' +
```

Comment:
I've removed the whitespaces on the empty lines, but it's still complaining; the `events.js` file it's talking about was not something I touched.